test/index.js

```diff
@@ -545,6 +545,28 @@ describe('def', function() {
                    'as its first argument; received ["abc"]'));
   });
 
+  it('supports "enum" types', function() {
+    var def = $.create($.env.concat([$.Enum, $.ValidDate, $.ValidNumber]));
+
+    //  convertTo :: Enum -> ValidDate -> ValidNumber
+    var convertTo =
+      def('convertTo',
```

Comment:
Nit: Could you nudge this line (and the following six lines) two spaces to the left? That's the indentation we're using elsewhere.